src/app/launcher/import-app/project-summary-importapp-step/project-summary-importapp-step.component.html

```diff
@@ -268,8 +268,8 @@ <h2 class="card-pf-title">
   <div class="container-fluid">
     <div id="ProjectSummary" class="f8launcher-continue">
       <button class="btn btn-primary btn-xlarge"
-              [disabled]="!completed"
-              (click)="setup()">Import Application</button>
+              [disabled]="(!completed && !setupInprogress) || (completed && setupInprogress)"
```

Comment:
This is mind confusing, can you really know what happens here? :p 
Could you please find a simpler expression if possible?

Reply:
I over think about this it can be simpler.